### PR TITLE
Case Showcase make close the primary action instead of cancel

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/overview/CaseOverviewViewImpl.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
                 <!-- actions -->
                 <div class="col-xs-12 col-sm-3 col-md-2 kie-case-view-header__actions">
       <span class="btn-group">
-          <button type="button" class="btn btn-primary" data-field="case-cancel" data-i18n-key="Cancel"></button>
+          <button type="button" class="btn btn-primary" data-field="case-destroy" data-i18n-key="Close"></button>
```
Comment (bsremac) [inline jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/overview/CaseOverviewViewImpl.html]: Is there a particular reason why in the action behind the button and data-field attribute don't match i.e. _Close_ vs. _destroy_?

Comment (cristianonicolai) [reply jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/overview/CaseOverviewViewImpl.html]: @bsremac aligned a few more things based on this rename for consistency.

**Comment by bsremac** [comment]
@cristianonicolai I'm not sure if you missed this comment of mine or you decided not to address it:
"_Does it make sense to swap the primary action for a case in case list as well?_"
If it's the latter, it's fine, I just want to know if it is based on a decision or on not being aware of it :)

**Comment by cristianonicolai** [comment]
@bsremac I was trying to get away :P sorry I missed it, fixing now...

**Comment by cristianonicolai** [comment]
@bsremac done.

**Pull request merged**